heroku/cli
# refactor(pg-v5): Move command pg:upgrade to oclif

### Description
https://gus.lightning.force.com/lightning/r/ADM_Work__c/a07EE00001bDLC6YAO/view

This converts `pg:upgrade` to oclif and updates its associated tests.

### Testing

1) Pull down this branch
2) Run `./bin/run pg:upgrade --app APP` with an app that has a postgres db that can be upgraded
3) Verify the command succeeds
4) Compare with prod
